toonz/sources/toonz/mainwindow.cpp

```diff
@@ -1603,6 +1603,7 @@ void MainWindow::defineActions() {
   createMenuFileAction(MI_Render, tr("&Render"), "Ctrl+Shift+R");
   createMenuFileAction(MI_FastRender, tr("&Fast Render to MP4"), "Alt+R");
   createMenuFileAction(MI_Preview, tr("&Preview"), "Ctrl+R");
+  createMenuFileAction(MI_SoundTrack, tr("&Export Soundtrack"), "Alt+S");
```

Comment:
Please set another default shortcut key since `Alt+S` is already set for `Startup Popup` .
Or you can just remove the default shortcut if this command seems to be triggered not so often.